Source/WebKit/UIProcess/API/Swift/DownloadCoordinator.swift

```diff
@@ -28,6 +28,9 @@ import Foundation
 // MARK: DownloadCoordinator protocol
 
 @_spi(Private)
+@available(WK_IOS_TBA, WK_MAC_TBA, WK_XROS_TBA, *)
+@available(watchOS, unavailable)
+@available(tvOS, unavailable)
```

Comment:
It's interesting that we have to add these at all, since this code is already guarded by `ENABLE_SWIFTUI`, which is based on the WK_ENABLE_SWIFTUI build setting that you've set based on deployment target. Still, it seems right to add availability annotations, since these are SPI.

cc @rr-codes